symphony/lib/core/class.errorhandler.php

```diff
@@ -417,7 +456,7 @@ public static function isEnabled()
      * @return string
      *  Usually a string of HTML that will displayed to a user
      */
-    public static function handler($code, $message, $file = null, $line = null)
+    public final static function handler($code, $message, $file = null, $line = null)
```

Comment:
[According to the PSR-2 spec](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#45-abstract-final-and-static)

> When present, the abstract and final declarations MUST precede the visibility declaration.
> When present, the static declaration MUST come after the visibility declaration.